hack/verify-govet.sh

```diff
@@ -23,7 +23,8 @@ FAILURE=false
 test_dirs=$(find_files | cut -d '/' -f 1-2 | sort -u)
 for test_dir in $test_dirs
 do
-  go tool vet -shadow=false $test_dir
+  # composites are disabled because fielderrors.ValidationErrorList{...} is incorrectly flagged
```

Comment:
I'm validating an array of `from` that is nested within another object, and applying a prefix (validateimagestreamimport) to the array.  No one else is really dealing with arrays in their tests.